### PR TITLE
Refine cd pipeline

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
#### Summary of Changes (What does this PR do?)
- [Seperates the (continous) deployment from the (continous) integration](https://github.com/airqo-platform/AirQo-frontend/commit/758efef275bde76d5f62475ee0a76664e3ce5949), in order to have one sole tool for that(ArgoCD)
- Use [netmanager and platform product names](https://github.com/airqo-platform/AirQo-frontend/commit/4160f33202ea22db08aea3344c56c199fd0b011b) instead of "platform" and "next platform" respectively

#### Status of maturity (all need to be checked before merging):
- [ ] I've tested this locally
- [x] I consider this code done
- [x] This change ready to hit production in its current state
- [x] The title of the PR states what changed and the related issues number (used for the release note).
- [ ] I've included issue number in the "Closes #ISSUE-NUMBER" part of the "[What are the relevant tickets?](#what-are-the-relevant-tickets)" section to [link the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
- [ ] I've updated corresponding documentation for the changes in this PR.
- [ ] I have written unit and/or e2e tests for my change(s).

#### How should this be manually tested?
- Do a static code review

### CODE REVIEW

File: .github/workflows/deploy-frontends-to-production.yml [hunk 1]
```diff
@@ -107,14 +107,10 @@ jobs:
...
+      - name: Update the corresponding configmap
         run: |
-          echo "=============== Update the corresponding configmap ==============="
-          kubectl create configmap --dry-run=client -o yaml --from-env-file=netmanager/.env env-platform-production | kubectl replace -f - -n production
-          echo "=============== Deploy the application ==============="
-          kubectl apply -f k8s/netmanager/prod-airqo-platform-frontend.yaml
-          kubectl set image deployment/airqo-platform-frontend airqo-platform=${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/airqo-platform-frontend:${{ needs.image-tag.outputs.build_id }} -n production
-          kubectl annotate deployment/airqo-platform-frontend kubernetes.io/change-cause="Image updated to ${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/airqo-platform-frontend:${{ needs.image-tag.outputs.build_id }} on ${{ needs.image-tag.outputs.datetime }}" -n production
+          kubectl create configmap --dry-run=client -o yaml \
+          --from-env-file=netmanager/.env env-platform-production | kubectl replace -f - -n production
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-frontends-to-production.yml:113]: [Seperate the (continous) deployment from the (continous) integration](https://github.com/airqo-platform/AirQo-frontend/pull/960/commits/758efef275bde76d5f62475ee0a76664e3ce5949)

File: .github/workflows/deploy-frontend-pr-previews.yml [hunk 2]
```diff
@@ -305,16 +305,16 @@ jobs:
...
 
-  next-platform-pr-comment:
-    name: next-platform-preview-link-comment
-    if: needs.check.outputs.run_next_platform == 'true'
-    needs: [next-platform]
+  analytics-platform-pr-comment:
+    name: analytics-platform-preview-link-comment
+    if: needs.check.outputs.run_analytics_platform == 'true'
+    needs: [analytics-platform]
     runs-on: ubuntu-latest
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-frontend-pr-previews.yml:318]: [netmanager and platform product names](https://github.com/airqo-platform/AirQo-frontend/pull/960/commits/4160f33202ea22db08aea3344c56c199fd0b011b)

File: .github/workflows/deploy-frontend-pr-previews.yml [hunk 3]
```diff
@@ -284,19 +284,19 @@ jobs:
...
       - name: Deploy to Cloud Run
         run: |-
-          gcloud run deploy ${{ needs.branch-name.outputs.lowercase }}-next-platform-preview \
+          gcloud run deploy ${{ needs.branch-name.outputs.lowercase }}-analytics-platform-preview \
             --region=${{ secrets.REGION }} \
             --max-instances=10 \
             --timeout=60 \
             --concurrency=10 \
-            --image=${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/pr-previews/next-platform-pr-previews:${{ github.sha }} \
+            --image=${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/pr-previews/analytics-platform-pr-previews:${{ github.sha }} \
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-frontend-pr-previews.yml:299]: [netmanager and platform preview names](https://github.com/airqo-platform/AirQo-frontend/pull/960/commits/bd4b918a8de20d158d33ca13c6aa69b967991b46)

**Review by Baalmart — APPROVED** [review]
This LGTM. thanks @123MwanjeMike . Much appreciated!

**Comment by Baalmart** [comment]
@123MwanjeMike , an emerging merge conflict that needs your attention.

**Comment by 123MwanjeMike** [comment]
> @123MwanjeMike , an emerging merge conflict that needs your attention.

Thanks @Baalmart , I have resolved these

**Review by NoahNsimbe — APPROVED** [review]
LGTM

**Review by Baalmart — APPROVED** [review]
LGTM

**Pull request merged**